lib/matplotlib/backends/backend_ps.py

```diff
@@ -505,12 +505,12 @@ def set_font(self, fontname, fontsize, store=True):
                 self.fontname = fontname
                 self.fontsize = fontsize
 
-    def create_hatch(self, hatch):
+    def create_hatch(self, hatch, gc=None):
```

Comment:
```suggestion
    def create_hatch(self, hatch, gc):
```

If `gc` is None this will fail so make it required.

Reply:
I suggest either this or the other three suggestions (pass in lw like the other backends rather than the whole gc).

Reply:
Passing in only linewidth seems like a good idea